lib/miq_expression.rb

```diff
@@ -943,9 +943,8 @@ def self.build_relats(model, parent = {}, seen = [])
 
       next if ref.macro == :belongs_to && model.name != parent[:root]
 
-      # REMOVE ME: workaround to temporarily exlude certain mdoels from the relationships
-      excluded_models = EXCLUDE_FROM_RELATS[model.name]
-      next if excluded_models && excluded_models.include?(assoc.to_s)
+      # REMOVE ME: workaround to temporarily exclude certain models from the relationships
```

Comment:
This says temporary it is a temporary workaround, but I have no idea when this workaround can be removed.

Do you know when it can be removed?

Reply:
I have no idea also, I can investigate it but for now I am just reducing condition and doing rubocop fix.